tests/openshift/openshift-tests.yml

```diff
@@ -3,7 +3,40 @@
   tasks:
   - name: Test cluster health reports OK
     uri:
-      url: https://ocp.customer2.cor00005.cna.ukcloud.com:8443/healthz/ping
+      url: https://ocp.customer1.cor00005.cna.ukcloud.com:8443/healthz/ping
```

Comment:
This isn't correct - it was correct for the specific environment we tested with but it either needs to reflect the pipeline url (customer2) or it needs to be parameterised to take the domain suffix from the ansible variables.